homeassistant/components/mealie/coordinator.py

```diff
@@ -82,16 +82,13 @@ class MealieMealplanCoordinator(
 
     async def _async_update_internal(self) -> dict[MealplanEntryType, list[Mealplan]]:
         next_week = dt_util.now() + WEEK
-        data = (
-            await self.client.get_mealplans(dt_util.now().date(), next_week.date())
-        ).items
+        current_date = dt_util.now().date()
+        next_week_date = next_week.date()
+        response = await self.client.get_mealplans(current_date, next_week_date)
         res: dict[MealplanEntryType, list[Mealplan]] = {
```

Comment:
type is likely redundant now and could be dropped since mypy can work it out from the dictcomp

Reply:
Oh, it did not, I added it because mypy wanted it